open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/toolbox/ControlPanelToolboxImpl.java

```diff
@@ -86,38 +72,37 @@ protected DetailPanelProviderRegistry createDetailPanelProviderRegistry(Toolbox
     public String getDescription()
     {
         return "A toolbox extension for control panels";
+
```

Comment:
Remove the extra blank line (you can get Eclipse to remove extra lines for you on save)